tests/types/test_component_gt.py

```diff
@@ -1,17 +1,18 @@
-"""Tests component.gt type, version 000"""
+"""Tests component.gt type, version 001"""
 
-from gwproto.data_classes.components import Component
 from gwproto.types import ComponentGt
-from tests.component_load_utils import ComponentCase, assert_component_load
 
 
 def test_component_gt_generated() -> None:
     d = {
-        "ComponentId": "987e0a5f-9036-411e-ba30-bac1075114ba",
-        "ComponentAttributeClassId": "cec0cb71-77bf-48a6-b644-2dcf124ac9fa",
-        "DisplayName": "Sample Component",
+        "ComponentId": "c6ec1ddb-5f51-4902-9807-a5ebc74d1102",
+        "ComponentAttributeClassId": "739a6e32-bb9c-43bc-a28d-fb61be665522",
+        "DisplayName": "Demo eGauge Power Meter",
         "HwUid": "000aaa",
         "TypeName": "component.gt",
         "Version": "000",
     }
-    assert_component_load([ComponentCase("ComponentGt", d, ComponentGt, Component)])
+
+    d2 = ComponentGt.model_validate(d).model_dump(exclude_none=True)
```

Comment:
We can add this to assert_component_load (and assert_cac_load) if we want.

Reply:
I didn't mean to take out the assert_component_load.  I am refactoring from the start again, setting up CACS_BY_MAKE_MODEL first to prepare for scada edits.  I've added it back in that one. I didn't add the second line to assert_component_load.